filebeat/docs/inputs/input-log.asciidoc

```diff
@@ -66,6 +114,9 @@ a pattern that matches the file you want to harvest and all of its rotated
 files. Also make sure your log rotation strategy prevents lost or duplicate
 messages. For more information, see <<file-log-rotation>>.
 
+Furthermore, do not use `path` method for `file_identity` to avoid duplication
+of the rotated log messages.
```

Comment:
```suggestion
Furthermore, to avoid duplication of rotated log messages,
do not use `path` method for `file_identity` 
```
this is slightly nicer.

Better still would be to suggest how to exclude the rotated paths.